boilerplate/openshift/golang-osd-operator/README.md

```diff
@@ -14,7 +14,7 @@ include boilerplate/generated-includes.mk
 
 ## Code coverage
 - A `codecov.sh` script, referenced by the `coverage` `make` target, to
-run code coverage analysis per [this SOP](https://github.com/openshift/ops-sop/blob/ff297220d1a6ac5d3199d242a1b55f0d4c433b87/services/codecov.md).
+run code coverage analysis per [this SOP](https://github.com/openshift/ops-sop/blob/93d100347746ce04ad552591136818f82043c648/services/codecov.md).
```

Comment:
Why not just `https://github.com/openshift/ops-sop/blob/master/services/codecov.md`?

Reply:
In case it moves.